packages/conversation/src/OpenAi.ts

```diff
@@ -12,13 +12,16 @@ function delay(ms: number) {
 
 export const DEFAULT_MODEL: TiktokenModel = 'gpt-3.5-turbo';
 export class OpenAi {
+  private static functionCallCount = 0;
```

Comment:
not static

Reply:
this is wrong all together. should not be an instance member either. needs to be recursively passed as a function argument to generateResponse